src/intervals/functions.jl

```diff
@@ -269,3 +269,106 @@ for f in (:log, :log2, :log10, :log1p)
 
         end
 end
+
+```
+This function takes an interval and a conversion specifier as input and gives a string interval containing the interval x.
+    for cs equal to u , it outputs empty, entire and nai in upper case
+    for cs equals to l, it outputs empty, entire and nai in lower case
+    for cs equal to f, it outputs full interval [-Inf,Inf] instead of entire interval.
+    for any other it outputs default string interval.
+```
+function string(x :: Interval{T}, cs :: Char) where T
+    if cs == 'u'
+        x == ∅ && return "[Empty]"
+        x == entireinterval(T) && return "Entire"
+        isnai(x) && return "[Nai]"
+    end
+    if cs == 'l'
+        x == ∅ && return "[empty]"
+        x == entireinterval(T) && return "entire"
+        isnai(x) && return "[nai]"
+    end
+    if cs == 'f'
+        x == entireinterval(T) && return "[-Inf,Inf]"
+    end
+    return string(x)
+end
+
+```
+Without any parameters, string function outputs the string of interval containing the interval x within 2 precision digits.
+julia> string(1.12321 .. 2.21322)
+"[1.12, 2.22]"
+```
+function string(x :: Interval{T}) where T
+    x == ∅ && return "[Empty]"
```

Comment:
I do not think that capitalized should be the default. Does the standard specify that?